source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/index.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+
+import { withTranslations, i18nProps } from '@lib/i18n';
+import { withLayout } from '@ui/components/layout';
+import Navigation from './navigation';
+import { Switch, Route } from 'react-router-dom';
+import OrganizationsRoute from '@ui/routes/admin/settings/organizations';
+
+export const pathName = '/admin/settings';
+
+type IProps =
+  & i18nProps;
+
+class AdminSettingsRoute extends React.Component<IProps> {
+  render() {
+    const { t } = this.props;
+    return (
+      <div className='ui container'>
```

Comment:
this layout and navigation should maybe wrap all of admin, and not just the settings, yeah?

Reply:
Yes, but haven't done that because we have inviteOrganization page that doesn't follow the super admin style.

Reply:
that's fine.
visual style doesn't matter.

structural style is what matters <3